www/docs/usage/commandline.md

```diff
@@ -53,45 +56,78 @@ The most recently selected versions are presented at the top of the dropdown.
 ## Show latest implicit version for stable releases
 
 1. Show the latest implicit stable version.
-2. Ex: `tfswitch -S 0.13` or `tfswitch --show-latest-stable 0.13` shows 0.13.6 (latest) version.
+2. Ex: `tfswitch -S 0.13` or `tfswitch --show-latest-stable 0.13` shows 0.13.6
```

Comment:
Is there a reason these are being split into new lines? Just linting, because it feels.. odd

Reply:
> Is there a reason these are being split into new lines?

Yes, the reason is to maintain sane line lengths.
E.g. https://google.github.io/styleguide/docguide/style.html#character-line-limit: «_Markdown content follows the residual convention of an 80-character line limit. Why? Because it’s what most of us do for code._»
Also:
- https://dsccommunity.org/styleguidelines/markdown-files/#line-length-not-longer-than-80-characters
- https://learn.microsoft.com/en-us/powershell/scripting/community/contributing/general-markdown?view=powershell-7.5#limit-line-length-to-100-characters
- 

> Just linting, because it feels.. odd

Well, for me the overly long lines feels odd 😺